neurokit2/eda/eda_process.py

```diff
@@ -64,6 +64,10 @@ def eda_process(eda_signal, sampling_rate=1000, method="neurokit"):
     >>> fig #doctest: +SKIP
 
     """
+    # Series check for non-default index
+    if type(eda_signal) is pd.Series and type(eda_signal.index) != pd.RangeIndex:
+        eda_signal.reset_index(inplace=True, drop=True)
```

Comment:
```suggestion
        eda_signal = eda_signal.reset_index(drop=True)
```

Thanks @csavur ! Maybe that'd be better because I remember that sometime ago in place modification modified also the object in the global environment (instead within the function's scope)